src/main/cli/settings.c

```diff
@@ -480,7 +480,7 @@ static const char * const lookupTableOffOnAuto[] = {
 };
 
 const char* const lookupTableInterpolatedSetpoint[] = {
-    "OFF", "ON", "AVERAGED_2", "AVERAGED_3", "AVERAGED_4"
+    "LEGACY", "NORMAL", "AVERAGED_2", "AVERAGED_3", "AVERAGED_4"
```

Comment:
Since the "LEGACY" mode is something to be avoided why can't we just make "OFF" mean off instead. off meaning no feed forward at all. I don't see why anyone would want to enable a feed forward mode that enables one of two outdated modes.

Reply:
It's tough to know what to do with the 'legacy' mode. If `legacy` is selected, and....
- the rc_interpolation mode is off, then FF becomes spikes unless heavily filtered
- the rc_interpolation mode is on, then setpoint is generally OK, and FF is usable, but both benefit from filtering.
I'll be making another PR that allows filtering in legacy mode, in which case both filtering and interpolation can be enabled or disabled independently.  We will need to do something (not exactly sure what) to disable feed forward if both are off.  
I don't think anyone has enabled `legacy` mode for some time.  Interpolation itself is currently not working properly, and results in a lot of sharp spikes.  It has no benefit over the step based method that I can see. Removing it would be quite a big job, but I think the new method has proven very effective, and if the old interpolation method was removed, the code would be simplified a lot.  And then we can just have a feedforward mode `OFF` that means, well, `OFF`.
So yeah, removing the old linear interpolation code makes a lot of sense.

Reply:
The other advantage of removing legacy interpolation mode, is that the `rc_smoothing_mode`, which currently moves between`filter` and `interpolation`, would simply become an `rc_filtering` on/off switch.